### ISSUE TITLE
Config Check: core directory and .htaccess

### ISSUE THREAD

**Issue by sebastian-marinescu** [modxcms/revolution#12918]
### Summary
Even after changing the .htaccess, so that it simply 'secures' the core-directory,
the warning message still exists.

### Step to reproduce
Install MODX, go to the Dashboard and follow the simple instructions.

### Observed behavior
The warning still exists, even the user did what MODX asked for.

### Expected behavior
I expect that after following this instruction, the warning disappears. 

Either it should really vanish after changing the .htaccess, 
or it should be made more clear that hardening MODX and 
setting up the core outside the web-root is a requirement.

If the first option is actually what MODX wants to tell me there, 
then this could be fix by either adding following in the .htaccess: 

```
<Files *.txt>
    Order Deny,Allow
    Deny from all
</Files>
```

Or by changing [config_check.inc.php](https://github.com/modxcms/revolution/blob/master/core/model/modx/processors/system/config_check.inc.php#L69) to actually try to access a php-file, 
which the .htaccess already covers.

### Environment
MODX 2.4.2pl

**Comment by Mark-H** [comment]
Should be fixed with the new .htaccess in 2.4.3 (or 2.5, can't remember)

**Comment by sebastian-marinescu** [comment]
Oh, I see. Thanks Mark! I'll close right after I updated and confirm this :+1: 
Should be this week.

**Comment by Mark-H** [comment]
Just confirmed, this is fixed in 2.5.0-rc1, not 2.4.3.
